Rules/Rules.csproj

```diff
@@ -53,6 +45,17 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net451'">
+    <PackageReference Include="System.Management.Automation" Version="6.0.0-alpha13" />
+  </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Management.Automation" Version="6.0.1.1" />
+    <PackageReference Include="Microsoft.Management.Infrastructure" Version="1.0.0-alpha08" />
```

Comment:
This is a good example to show that the old `System.Management.Automation` package was self-contained and now we also need its other dependency `Microsoft.Management.Infrastructure`.